cli/dcoscli/job/main.py

```diff
@@ -228,6 +233,39 @@ def _remove(job_id, stop_current_job_runs=False):
     return 0
 
 
+def _queue(job_id, json_flag=False, quiet=False):
+    """
+    :param job_id: Id of the job
+    :type job_id: str
+    :param json_flag: output json if True
+    :type json_flag: bool
+    :returns: process return code
+    :rtype: int
+    """
+    client = metronome.create_client()
+    deployment_list = client.get_queued_jobruns(job_id)
+
+    if not deployment_list and not json_flag:
+        msg = "There are no deployments in the queue"
+        if job_id:
+            msg += " for '{}'".format(job_id)
+            raise DCOSException(msg)
+        else:
```

Comment:
Why is just `msg` published and it returns? Why depends on `job_id`?

Reply:
I am actually a little bit lost here too...

Reply:
Maybe it is error when asking for concrete job id but not an error in general? But I would say it should not raise exception in both cases...

Reply:
explanation provided above... the job_id is added the msg if possible to provide the most information

Reply:
I still don't agree that when I ask for a queue of specific job that it is error when that job is not in the queue - I would expect to get an empty result instead if it is not in the queue...